### PR TITLE
MTV-1446: MTV 2.6.7 release notes

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1446](https://issues.redhat.com/browse/MTV-1446) - Release notes

* [MTV-1447](https://issues.redhat.com/browse/MTV-1447) - Attributes

### PREVIEW

* [MTV 2.6.7 resolved issues](https://anarnold97.github.io/MTA-Preview/MTV/rn-2-6-7.html#resolved-issues-26-7_release-notes)

### CODE REVIEW

File: documentation/modules/rn-2.6.adoc [hunk 1]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-7_{context}"]
+=== Resolved issues 2.6.7
+
+.Incorrect handling of double quotes in ifcfg files
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:147]: "handling of quotes"

File: documentation/modules/rn-2.6.adoc [hunk 2]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
...
+
+In earlier releases of {project-short}, there was an issue with the incorrect handling of double quotes in interface configuration (ifcfg) files, which control the software interfaces for individual network devices. This issue has been resolved in {project-short} 2.6.7, in order to cover additional IP configurations on Red Hat Enterprise Linux, CentOS, Rocky Linux and similar distributions. link:https://issues.redhat.com/browse/MTV-1439[(MTV-1439)]
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:149]: "handling of single and double quotes"

File: documentation/modules/rn-2.6.adoc [hunk 3]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
...
+
+.Failure to preserve netplan based network configuration
+
+In earlier releases of {project-short}, there was an issue with the preservation of netplan-based network configurations. This issue has been resolved in {project-short} 2.6.7, so that static IP configurations are preserved if netplan (netplan.io) is used by using the netplan configuration files to generate udev rules for known mac-address and `ifname` tuples. link:https://issues.redhat.com/browse/MTV-1440[(MTV-1440)]
+
+.Error messages corrupt in udev .rules files
```
Comment (fabiand) [inline documentation/modules/rn-2.6.adoc:155]: "Error messages got written into udev .rules files"

**Review by fabiand — COMMENTED** [review]
small nits

File: documentation/modules/rn-2.6.adoc [hunk 4]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
 
 This release has the following resolved issues:
 
+[id="resolved-issues-26-7_{context}"]
+=== Resolved issues 2.6.7
+
+.Incorrect handling of quotes in ifcfg files
+
+In earlier releases of {project-short}, there was an issue with the incorrect handling of double quotes in interface configuration (ifcfg) files, which control the software interfaces for individual network devices. This issue has been resolved in {project-short} 2.6.7, in order to cover additional IP configurations on Red Hat Enterprise Linux, CentOS, Rocky Linux and similar distributions. link:https://issues.redhat.com/browse/MTV-1439[(MTV-1439)]
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:149]:
```suggestion
In earlier releases of {project-short}, there was an issue with the incorrect handling of single and double quotes in interface configuration (ifcfg) files, which control the software interfaces for individual network devices. This issue has been resolved in {project-short} 2.6.7, in order to cover additional IP configurations on Red Hat Enterprise Linux, CentOS, Rocky Linux and similar distributions. link:https://issues.redhat.com/browse/MTV-1439[(MTV-1439)]
```

File: documentation/modules/rn-2.6.adoc [hunk 5]
```diff
@@ -141,6 +141,21 @@ The validation service includes default validation rules for virtual machines fr
...
+
+.Incorrect handling of quotes in ifcfg files
+
+In earlier releases of {project-short}, there was an issue with the incorrect handling of single and double quotes in interface configuration (ifcfg) files, which control the software interfaces for individual network devices. This issue has been resolved in {project-short} 2.6.7, in order to cover additional IP configurations on Red Hat Enterprise Linux, CentOS, Rocky Linux and similar distributions. link:https://issues.redhat.com/browse/MTV-1439[(MTV-1439)]
+
+.Failure to preserve netplan based network configuration
+
+In earlier releases of {project-short}, there was an issue with the preservation of netplan-based network configurations. This issue has been resolved in {project-short} 2.6.7, so that static IP configurations are preserved if netplan (netplan.io) is used by using the netplan configuration files to generate udev rules for known mac-address and `ifname` tuples. link:https://issues.redhat.com/browse/MTV-1440[(MTV-1440)]
+
+.Error messages corrupt in udev .rules files
```
Comment (anarnold97) [inline documentation/modules/rn-2.6.adoc:155]:
```suggestion
.Error messages are written into udev .rules files
```

**Review by fabiand — APPROVED** [review]
looks great to me

**Review by fabiand — APPROVED** [review]
looks great to me, please merge.

**Pull request merged**